cartographer/transform/transform.h

```diff
@@ -51,7 +51,8 @@ T GetYaw(const Rigid3<T>& transform) {
 // rotation as the given 'quaternion'.
 template <typename T>
 Eigen::Matrix<T, 3, 1> RotationQuaternionToAngleAxisVector(
-    Eigen::Quaternion<T> quaternion) {
+    const Eigen::Quaternion<T>& _quaternion) {
```

Comment:
This code base follows the [Google C++ style](https://google.github.io/styleguide/cppguide.html), formatting is easiest done by installing clang-format and invoking, e.g., `clang-format -i -style=Google cartographer/transform/transform.h`.

Here: Revert the argument name to 'quaternion', and rather make the local name more descriptive, e.g.:
`const Eigen::Quaternion<T> normalized = quaternion.normalized();`